components/TicketsSection.js

```diff
@@ -5,7 +5,7 @@ export default function TicketsSection () {
     <div className='tickets'>
       <SectionTitle>Tickets</SectionTitle>
       <TicketsTable />
-      <Button href='https://www.eventpop.me/e/1809-react-bangkok-2-0-0'>Get Tickets</Button>
+      <Button href='https://www.eventpop.me/e/1809-react-bangkok-2-0-0' isFull>Get Tickets</Button>
```

Comment:
More proper props would be `disabled` because it resemble `<button>`'s attribute and it can be reused anywhere else.